src/apps/sketch/communication/messageFormat.js

```diff
@@ -0,0 +1,104 @@
+// all game logic goes here
+export class MessageFormat {
```

Comment:
These messages handlers should also handle routing, and logic implementation, they should know how to read a message, we only pass it message data. In memory database type logic should also be there